projects/ngx-cookie-service/src/lib/cookie.service.ts

```diff
@@ -125,7 +125,13 @@ export class CookieService {
       cookieString += 'domain=' + domain + ';';
     }
 
-    if ( secure ) {
+    if ( secure === false && sameSite === 'None') {
```

Comment:
to account for cases when secure is undefined, should we perform a falsy check on secure ie. if (!secure), instead of strict boolean check?

or alternatively set the default to be `false` and it could prevent an undefined case

Reply:
@brandontulsi feel free to do a PR, you are right